x-pack/legacy/plugins/canvas/public/lib/notify.js

```diff
@@ -5,16 +5,13 @@
  */
 
 import { get } from 'lodash';
-import { getCoreStart, getStartPlugins } from '../legacy';
+import { getCoreStart } from '../legacy';
```

Comment:
So does this file get deleted with the services rework? In favor of, `x-pack/legacy/plugins/canvas/public/services/notify.ts` ?